inverse/beamformer_dics.m

```diff
@@ -526,6 +526,9 @@
     elseif isstruct(refdip) && isfield(refdip, 'pos') % check if only position of refdip is present
       assert(isnumeric(refdip.pos) && numel(refdip.pos)==3);
       lf1 = ft_compute_leadfield(refdip.pos, grad, headmodel, 'reducerank', reducerank, 'normalize', normalize);
+      if isfield(refdip,'mom'); % check for fixed orientation
+        lf1 = lf1.*refdip.mom'; 
```

Comment:
if you multiply with refdip.mom(:), then you know for sure that it is a column vector, regardless of the input.